build/nlu-regression/index.js

```diff
@@ -46,30 +20,34 @@ async function runTest(test, { update, keepGoing }) {
   const previousPerformance = await readResults(name)
   const comparison = evaluatePerformance(performance, previousPerformance)
 
-  const regressionMessage = `\n${formatRegressionMessage(name, comparison, performance.groupedBy)}\n`
+  bitfan.visualisation.showComparisonReport(name, comparison)
+  console.log("")
```

Comment:
useful ?

Reply:
yeah, its to space out the output. lol, I could have added a `\n` instead...